<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_criteria</class>
 <widget class="QWidget" name="add_criteria">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>准则</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,2,3">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>主要准则</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QListWidget" name="listWidget">
          <item>
           <property name="text">
            <string>FFT</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>LSC</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MIC</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MCC</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MZC</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>FLT</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MDi</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mde</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MDw</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MS3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MRS</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>TSL</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>WTH</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>次要准则</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QListWidget" name="listWidget_2">
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>选择滤波器</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QTableWidget" name="tableWidget">
            <row>
             <property name="text">
              <string>FL0</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>FL1</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>FL2</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>FL3</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>FL4</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>FL5</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>FL6</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>FL7</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>FL8</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>FL9</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>自</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>至</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>调谐指数</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QSpinBox" name="spinBox"/>
          </item>
          <item row="1" column="0">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>162</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="confirm_pushButton">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
